class/DatabaseConnection.php

```diff
@@ -0,0 +1,336 @@
+	{
+		if($table_name == 'address')
+		{
+			$del_add = "DELETE FROM $table_name WHERE emp_id = " . $id;
+			DatabaseConnection::db_query($del_add);
+		}		
+		elseif($table_name == 'employee')
+		{
+			$del_emp = "DELETE FROM $table_name WHERE id = " . $id;
+			DatabaseConnection::db_query($del_emp);
+		}
+	}
+
+	/**
+	* To delete the employee profile picture
+	*
+	* @access public
+	* @param  integer $id
+	* @return void
+	*/
+	public function delete_pic($id)
+	{
+		$sel_pic = "SELECT photo FROM employee WHERE id = " . $id;
+		$pic_object = DatabaseConnection::db_query($sel_pic);
+		$row = DatabaseConnection::db_fetch_array($pic_object);
+		$pic_location = PIC_PATH . $row['photo'];
+		unlink($pic_location);
+	}
+
+	/**
+	* To display the employee list
+	*
+	* @access public
+	* @param  void
+	* @return object
+	*/
+	public function display_list()
+	{
+		$q_list = "SELECT emp.prefix AS prefix, 
+			CONCAT(emp.first_name,' ', emp.middle_name,' ', emp.last_name) AS name, 
+			emp.gender AS gender, emp.dob AS dob, 
+			emp.marital_status AS marital_status, emp.employment AS employment, 
+			emp.employer AS employer, 
+			CONCAT(res.street, ', ', res.city, ', ', res.state, ', ', res.zip, ', ', 
+			res.phone, ', ', res.fax) AS res_add, 
+			CONCAT(off.street, ', ', off.city, ', ', off.state, ', ', off.zip, ', ', 
+			off.phone, ', ', off.fax) AS off_add, emp.comm_id AS comm_id, emp.id AS id, emp.photo AS photo
+			from employee AS emp 
+			inner join address AS res on (emp.id = res.emp_id and res.address_type = 'residence')
+			inner join address AS off on (emp.id = off.emp_id and off.address_type = 'office')";
+
+		$result_list = DatabaseConnection::db_query($q_list);
+		return $result_list;
+	}
+
+	/**
+	* To display the employee list
+	*
+	* @access public
+	* @param  string $table_name, array $data_array
+	* @return void
+	*/
+	public function update_table($table_name, $data_array)
+	{ 
+		if($table_name == 'address')
+		{
+			$update_add_res = "UPDATE $table_name 
+				SET `street` = '{$data_array['r_street']}', `city` = '{$data_array['r_city']}', 
+				`state` = '{$data_array['r_state']}', `zip` = {$data_array['r_zip']}, 
+				`phone` = {$data_array['r_phone']}, `fax` = {$data_array['r_fax']} 
+				WHERE (address_type = 'residence' AND emp_id = {$data_array['emp_id']})";
+
+			DatabaseConnection::db_query($update_add_res);
+
+			$update_add_off = "UPDATE $table_name 
+				SET `street` = '{$data_array['o_street']}', `city` = '{$data_array['o_city']}', 
+				`state` = '{$data_array['o_state']}', `zip` = {$data_array['o_zip']}, 
+				`phone` = {$data_array['o_phone']}, `fax` = {$data_array['o_fax']} 
+				WHERE (address_type = 'office' AND emp_id = {$data_array['emp_id']})";
+
+			DatabaseConnection::db_query($update_add_off);
+		}
+		elseif($table_name == 'employee')
+		{
+			if(isset($data_array['photo']))
+			{
+				$update_emp = "UPDATE $table_name 
+					SET `first_name` = '{$data_array['first_name']}', 
```

Comment:
Can define common set values in one variable and can use that variable in both cases